core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChildSet.java

```diff
@@ -0,0 +1,139 @@
+package nl.nn.adapterframework.doc.model;
```

Comment:
License info

Reply:
Gedaan. Maar ik heb niet gekeken of ik "2020, 2021" of alleen "2021" moet neerzetten.